### PR TITLE
Add S3 upload and download scripts.

### PR DESCRIPTION
**Pull request by orionr**
* Synchronizes models stored in GitHub LFS (used as system of truth) and Amazon S3 (cache used for general model downloads)
* Before I run these I'll selectively make sure the two locations are in sync. I'll be adding a lot of style transfer models, for example.

### CODE REVIEW

File: scripts/download_models_from_s3.sh [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+            ;;
+        *)
+            echo "Invalid option: $1"
+            exit 1
+            ;;
+    esac
+    shift
+done
+
+function sync_files {
```
Comment (houseroad) [inline scripts/download_models_from_s3.sh]: call it download_files?

File: scripts/upload_models_to_s3.sh [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+            ;;
+        *)
+            echo "Invalid option: $1"
+            exit 1
+            ;;
+    esac
+    shift
+done
+
+function sync_files {
```
Comment (houseroad) [inline scripts/upload_models_to_s3.sh]: call it upload_files?

File: scripts/download_models_from_s3.sh (hunk 1 above)
Comment (orionr) [reply scripts/download_models_from_s3.sh]: Sounds good!

File: scripts/upload_models_to_s3.sh (hunk 2 above)
Comment (orionr) [reply scripts/upload_models_to_s3.sh]: Same

**Comment by orionr** [comment]
Okay - added some nice comments to both files. Merging.

**Pull request merged**